### ISSUE TITLE
Restore and activate existing instance if it's already running

### ISSUE THREAD

**Issue by ngreyling** [handruin/spdif-ka#29]
Feature idea: If the application is already running when mistakenly trying to launch a second instance, restore and activate the window of the first instance.

**Comment by mayerwin** [comment]
Yes, except if Start Minimized is checked (it should not show the window). At least for now, it will not run a second instance, which is the most important.
